src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -27,3 +35,12 @@ def load_command_table(self, _):
                          confirmation='Kubernetes may be unavailable during cluster upgrades.\n' +
                          'Are you sure you want to perform this operation?')
         g.wait_command('wait')
+
+    # AKS agent pool commands
+    with self.command_group('aks nodepool', agent_pools_sdk, client_factory=cf_agent_pools) as g:
+        g.custom_command('list', 'aks_agentpool_list')
+        g.custom_show_command('show', 'aks_agentpool_show', table_transformer=aks_agentpool_show_table_format)
```

Comment:
Is table formatter the default one ? Or json formatter ?

Reply:
json formatter is the default one.